app/controllers/admin/api/services/backend_api_configs_controller.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+class Admin::Api::Services::BackendApiConfigsController < Admin::Api::Services::BaseController
+  self.access_token_scopes = :account_management
+
+  before_action :authorize
+
+  clear_respond_to
+  respond_to :json
+
+  wrap_parameters BackendApiConfig
+  representer BackendApiConfig
+
+  paginate only: :index
+
+  # swagger
+  ##~ sapi = source2swagger.namespace("Account Management API")
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/services/{service_id}/backend_apis.json"
+  ##~ e.responseClass = "List[backend_api_config]"
+  #
+  ##~ op            = e.operations.add
+  ##~ op.httpMethod = "GET"
+  ##~ op.summary    = "Backend Usage List"
+  ##~ op.description = "Returns the list of all backend being used by a service (product) with the corresponding path."
+  ##~ op.group = "backend_api_config"
+  #
+  ##~ op.parameters.add @parameter_access_token
+  ##~ op.parameters.add @parameter_service_id_by_id_name
+  #
+  def index
+    backend_api_configs = service.backend_api_configs.accessible.order(:id).paginate(pagination_params)
+    respond_with(backend_api_configs)
+  end
+
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/services/{service_id}/backend_apis.json"
+  ##~ e.responseClass = "backend_api_config"
+  #
+  ##~ op            = e.operations.add
+  ##~ op.httpMethod = "POST"
+  ##~ op.summary    = "Backend Usage Create"
+  ##~ op.description = "Adds a backend to a service (product)."
+  ##~ op.group = "backend_api_config"
+  #
+  ##~ op.parameters.add @parameter_access_token
+  ##~ op.parameters.add @parameter_service_id_by_id_name
+  ##~ op.parameters.add :name => "backend_api_id", :description => "Backend to be added to the service (product).", :dataType => "string", :required => false, :paramType => "query"
+  ##~ op.parameters.add :name => "path", :description => "Path of the association.", :dataType => "int", :required => false, :paramType => "query"
```

Comment:
Sorry, I missed this one.

```suggestion
  ##~ op.parameters.add :name => "path", :description => "Path of the backend for this product.", :dataType => "int", :required => false, :paramType => "query"
```